patches/chrome-browser-ui-cocoa-bookmarks-bookmark_menu_bridge.mm.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/cocoa/bookmarks/bookmark_menu_bridge.mm b/chrome/browser/ui/cocoa/bookmarks/bookmark_menu_bridge.mm
+index eed94ff1c8ef21522eadc73b4e976f53ed94dedd..acc16976767968dfd549006748c2713b6cbde995 100644
+--- a/chrome/browser/ui/cocoa/bookmarks/bookmark_menu_bridge.mm
++++ b/chrome/browser/ui/cocoa/bookmarks/bookmark_menu_bridge.mm
+@@ -126,7 +126,7 @@ void BookmarkMenuBridge::BuildRootMenu() {
+ 
+   // If the "Other Bookmarks" folder has any content, make a submenu for it and
+   // fill it in.
+-  if (!model->other_node()->children().empty()) {
++  BRAVE_BUILD_ROOT_MENU
```

Comment:
why is this needed? The other node will always be empty now, right?

Reply:
not always, chromium sync uses it as a temporary place to put foster children
https://cs.chromium.org/chromium/src/components/sync_bookmarks/bookmark_change_processor.cc?l=619&rcl=74cefac725837520ab3d553eb2fa531e5ee02f63

Reply:
addressed in 3f388ed041368c0b2b4911de083aca3733d4c775